pkg/agent/endpoints/workload/handler_test.go

```diff
@@ -6,21 +6,21 @@ import (
 	"crypto/rand"
 	"crypto/x509"
 	"errors"
-	"sort"
+	"github.com/spiffe/spire/pkg/agent/manager/cache"
```

Comment:
nit: this import belongs with the other github imports below

Reply:
Totally agree.